src/main/paradox/features/status.md

```diff
@@ -0,0 +1,42 @@
+## Status
```

Comment:
Healthchecks?

Reply:
Health and ready checks?

Reply:
Application Health, according to OpenShift - https://docs.openshift.org/latest/dev_guide/application_health.html

Reply:
also good name